tools/codegen/gofilegen/resource/resource_file_test.go

```diff
@@ -3,9 +3,9 @@ package resource_test
 import (
 	"testing"
 
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/common/autogen/stringcase"
 	"github.com/mongodb/terraform-provider-mongodbatlas/tools/codegen/codespec"
 	"github.com/mongodb/terraform-provider-mongodbatlas/tools/codegen/gofilegen/resource"
-	"github.com/mongodb/terraform-provider-mongodbatlas/tools/codegen/stringcase"
```

Comment:
Can we add a unit test here defining a model that does not have the delete operation?